src/Message/FundingRequest.php

```diff
@@ -18,21 +18,7 @@ public function getData()
 
     public function sendData($data)
     {
-        $response = $this->httpClient->request('GET', $this->getEndpoint($data), $this->getHeaders());
```

Comment:
For both the funding and inquire methods, we want the entire response to return, not a transformed version.